src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -288,7 +288,8 @@ private static Uri CreateConnectUrl(Uri url, NegotiationResponse negotiationResp
                         {
                             _logger.LogDebug("Raising Received event.");
 
-                            Received?.Invoke(buffer);
+                            // TODO: Return this
```

Comment:
This PR stomps on @mikaelm12 's PR which does this. Returning this requires more test changes.